templates/Blocks/_infoBox.html.twig

```diff
@@ -160,19 +160,19 @@
         <div class="info-box__item__content">
             <select id="version-select" class="select-color-green">
                 <option value="//demo.sylius.com">
-                    Sylius Demo (with Plus modules)
+                    Sylius 1.x - Plus demo
```

Comment:
```suggestion
                    Sylius 1.x - Plus Demo
```
I know it was specified this way in the task, but I would suggest sticking with capital letters across all entries for the sake of consistency.